ci-operator/step-registry/openshift/microshift/e2e/metal-tests/openshift-microshift-e2e-metal-tests-commands.sh

```diff
@@ -49,9 +49,18 @@ EOF
     cat >>${REPORT} <<EOF
     <p>${testname}:&nbsp;
     <a target="_blank" href="${URL}/${testname}">directory</a>
+EOF
+    if [ -f "${URL}/${testname}/boot_and_run.log" ]; then
```

Comment:
I don't think it'll work - you'd be checking remote file that is probably not yet created because the step is still running.
We need to check local path, so something like `${test}/boot_and_run.log` should work (`$test` should point to a specific scenario dir)

Reply:
Done